src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,10 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        Ball[] balls = {lottery.getRandomBall(),lottery.getRandomBall(),lottery.getRandomBall()};
```

Comment:
let's create and print balls directly in the loop, as stated in common mistakes :)